bin/codemods/src/assign-component-to-displayname-variable.js

```diff
@@ -50,91 +39,73 @@ const displayNamePropertyIdentifier = {
 
 const isValidComponentString = string => string === 'Component' || string === 'PureComponent';
 
-const argIsCreateClassInstance = arg =>
-	arg.get( 'callee', 'object', 'name' ).value === 'React' &&
-	arg.get( 'callee', 'property', 'name' ).value === 'createClass';
-
-const argIsExtendsComponentInstance = arg =>
-	arg.get( 'type' ).value === 'ClassExpression' &&
-	( isValidComponentString( arg.get( 'superClass', 'name' ).value ) ||
-		( arg.get( 'superClass', 'object', 'name' ).value === 'React' &&
-			isValidComponentString( arg.get( 'superClass', 'property', 'name' ).value ) ) );
-
-const argIsComponent = arg =>
-	argIsCreateClassInstance( arg ) || argIsExtendsComponentInstance( arg );
-
-const getMatchingArg = args => args.filter( argIsComponent )[ 0 ];
+const isCreateClassComponent = arg =>
```

Comment:
👍 to this name change too.
My only qualm with these renames is that they're quite generic, yet wouldn't apply generically (only to an argument). I don't think that's really an issue here, there's plenty of context and the argument signature gives clarity anyway :)